src/Apps/Order/Mutations/SetPaymentByStripeIntent.ts

```diff
@@ -0,0 +1,52 @@
+import { commitMutation, Environment, graphql } from "relay-runtime"
+import {
+  SetPaymentByStripeIntentMutation,
+  CommerceSetPaymentByStripeIntentInput,
+  SetPaymentByStripeIntentMutationResponse,
+} from "__generated__/SetPaymentByStripeIntentMutation.graphql"
+
+export const SetPaymentByStripeIntent = (
```

Comment:
I've flagged this before, but want to flag again: We really need to stop writing custom mutations every time we mutate! Please use the `useMutation` hook as [seen here](https://github.com/artsy/force/blob/main/src/Apps/Settings/Routes/DeleteAccount/useDeleteAccount.ts)

Reply:
this is my bad to forget to delete it! I actually have it [with useMutation](https://github.com/artsy/force/blob/main/src/Apps/Order/Mutations/useSetPaymentByStripeIntentMutation.ts) and we are [using that](https://github.com/artsy/force/blob/main/src/Apps/Order/Hooks/useStripePaymentBySetupIntentId.tsx#L11) one. 

This custom mutation is not used at all, so I'll just get rid of it